Automattic/wp-calypso
# Recent Feed Overhaul: Add engagement bar

Related to https://github.com/Automattic/loop/issues/195

## Proposed Changes

* This PR reuses the `ReaderPostActions` component to create a new `EngagementBar` that we use on the `Recent` posts view. The `EngagementBar` contains the same features as the "Post's Actions."
* The `EngagementBar` is fixed at the bottom of the post screen and visible when the "Post Actions" are off-screen. When the "Post Actions" are in the viewport, the `EngagementBar` is hidden.

Demo

https://github.com/user-attachments/assets/57c589e8-7b54-4cd9-b343-293086e7a122



## Why are these changes being made?


* To encourage user engagement with the post.

## Testing Instructions



* Use Calypso Live and go to /read?flags=reader/recent-feed-overhaul
* Interact with the engagement bar in various posts and test that it is working. Try to break it.
* Check the engagement bar in various widths and mobile view.

## Pre-merge Checklist
